src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -268,9 +268,8 @@ public function configureOptions(OptionsResolver $resolver)
     /**
      * Return the default loader object.
      *
-     * @param ObjectManager $manager
-     * @param mixed         $queryBuilder
-     * @param string        $class
+     * @param mixed  $queryBuilder
```

Comment:
mixed params could be removed as well as `mixed` does not give any useful information.

Reply:
I'd rather keep this, since `mixed` type is comming to PHP and it would make easier transition to scalar type, as in `string`,  `int` etc. before